bench/db_bench.ml

```diff
@@ -291,12 +312,12 @@ module Lmdb = struct
               Lmdb.put_string txn ddb k v >>= fun () -> sync env))
         ls
     in
-    print_results (write rw env random) nb_entries;
-    closedir env
+    closedir env;
```

Comment:
Here you can't reverse the two either, as `print_results (write rw env ...)` uses `env`